spec/support/user_helper.rb

```diff
@@ -19,7 +19,7 @@ def visit_create_card_path
 def create_card
   visit_create_card_path
   card_fill_in
-  select deck.name, from: 'Deck'
+  select deck.name, from: 'Имя колоды'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.